pages/linux/iostat.md

```diff
@@ -18,6 +22,6 @@
 
 `iostat -xN {{sda}}`
 
-- Display incremental reports of CPU and disk statistics every "interval" second(s) with units converted to megabytes:
+- Display incremental reports of CPU and disk statistics every "2" second(s):
```

Comment:
I'd remove the quote marks around "2" and the parenthesis around the s in second(s). These make sense for a generic example, but are awkward for a concrete one.